src/Game.h

```diff
@@ -98,6 +98,17 @@ class Game {
 
 	void SetTimeAccel(TimeAccel t);
 	void RequestTimeAccel(TimeAccel t, bool force = false);
+    
```

Comment:
I'd prefer it if was updated in the current commit, if possible (I'm in IRC if you need help).
http://pioneerwiki.com/wiki/Using_git_and_GitHub#Fixing.2FUpdating_your_pull_request